docs/reference/indices/rollover-index.asciidoc

```diff
@@ -42,19 +42,34 @@ NOTE: To roll over an index, a condition must be met *when you call the API*.
 automatically roll over indices when a condition is met, you can use {es}'s
 <<index-lifecycle-management, index lifecycle management (ILM) policies>>.
 
-The API accepts a single alias name and a list of `conditions`. The alias must point to a write index for
-a Rollover request to be valid. There are two ways this can be achieved, and depending on the configuration, the
-alias metadata will be updated differently. The two scenarios are as follows:
-
- - The alias only points to a single index with `is_write_index` not configured (defaults to `null`).
-
-In this scenario, the original index will have their rollover alias will be added to the newly created index, and removed
-from the original (rolled-over) index.
-
- - The alias points to one or more indices with `is_write_index` set to `true` on the index to be rolled over (the write index).
-
-In this scenario, the write index will have its rollover alias' `is_write_index` set to `false`, while the newly created index
-will now have the rollover alias pointing to it as the write index with `is_write_index` as `true`.
+The rollover index API accepts a single alias name
+and a list of `conditions`.
+To request a rollover,
+the specified alias must point to a write index,
+indicated by the `is_write_index` value.
+Depending on this value,
+the rollover request updates the alias metadata as follows:
+
+. The alias points to a single index
+with an unconfigured `is_write_index` value,
+which defaults to `null`.
++
+In this scenario,
+the rollover request adds the alias
+to the newly created target index
+and removes the alias
+from the rolled over index.
+
+. The alias points to one or more indices
+with an `is_write_index` value of `true`.
```

Comment:
I am not sure how to best describe this, but this scenario describes a situation where there are multiple indices with the same alias that is being rolled over. In this situation, exactly one of those indices needs to have their alias configured with `is_write_index` set to `true`, so that the rollover api treats that index as the writing index that is to be rolled over.

...

```
 The alias points to one or more indices with exactly one index with an `is_write_index` value of `true`
```
...

what do you think?

Reply:
Thanks for the clarification @talevy.

I made some changes with 6750300 and 3bf2b0f that address your feedback. Let me know if any further changes are needed.